src/app/canvas/Canvas.tsx

```diff
@@ -0,0 +1,128 @@
+'use client';
+
+import { useState, useCallback } from 'react';
+import update from 'immutability-helper';
+import { Stack } from '@mui/material';
+import { TaskCard } from '@/components/cards/TaskCard';
+import { TaskColumn } from '@/components/cards/TaskColumn';
+import { Droppable } from '@/hoc/Droppable';
+
+interface Task {
+  assignedTo: string;
+  id: number;
+}
+
+interface Lists {
+  [key: number]: Task[];
+}
+
+const mockTaskTodoData: Task[] = [
+  {
+    assignedTo: 'ragnar',
+    id: 1,
+  },
+  {
+    assignedTo: 'john',
+    id: 2,
+  },
+  {
+    assignedTo: 'yuki',
+    id: 3,
+  },
+];
+
+const mockTaskInProgressData: Task[] = [
+  {
+    assignedTo: 'doe',
+    id: 4,
+  },
+  {
+    assignedTo: 'floki',
+    id: 5,
+  },
+  {
+    assignedTo: 'professor',
+    id: 6,
+  },
+];
+
+const mockCompletedTask: Task[] = [
+  {
+    assignedTo: 'rock',
+    id: 7,
+  },
+];
+
+export const Canvas = () => {
```

Comment:
why is this component called a Canvas? Is there a reason its named generically or is there more descriptive name we can give it.

Reply:
Canvas was just a name given to the component. Renamed the component and filename from `Canvas` to `TaskBoard` which sounds more descriptive IMO.